apps/root/src/pages/strategy-guardian-detail/strategy-management/components/expected-returns-changes-summary/index.tsx

```diff
@@ -90,27 +138,33 @@ const ExpectedReturnsChangesSummary = ({
   const hasNewValues = !!updatedUserPositions && updatedUserPositions.length > 0;
   return (
     <ContainerBox gap={size === 'medium' ? 16 : 6} flexWrap="wrap">
-      {STRATEGY_RETURN_PERIODS.filter((period) => !hidePeriods?.includes(period.period)).map((period) => (
-        <ContainerBox flexDirection="column" key={period.period} gap={size === 'medium' ? 0 : 1}>
-          <Typography variant="bodySmallRegular">{intl.formatMessage(period.title)}</Typography>
-          {earnings[period.period] === updatedEarnings[period.period] ? (
-            <StyledCurrentValueBold>-</StyledCurrentValueBold>
-          ) : (
-            <NetWorthNumber
-              value={hasOriginalValue ? earnings[period.period] : updatedEarnings[period.period]}
-              isLoading={isLoading}
-              variant={hasOriginalValue && hasNewValues ? 'bodyRegular' : 'bodyBold'}
-            />
-          )}
-          {earnings[period.period] !== updatedEarnings[period.period] && hasOriginalValue && hasNewValues && (
-            <>
-              <StyledArrowIcon />
-              <ContainerBox gap={0.5} alignItems="center">
-                <NetWorthNumber value={updatedEarnings[period.period]} isLoading={isLoading} variant="bodyBold" />
-              </ContainerBox>
-            </>
+      {showTotal && (
+        <SummaryItem
```

Comment:
when does this appear?

Reply:
This field is only displayed on withdraw, I wanted to make it more flexible, not attaching it to variant="withdraw", but I'm thinking is better to attach it for now
<img width="427" alt="Screenshot 2024-09-09 at 3 31 58 PM" src="https://github.com/user-attachments/assets/927a2459-fce5-4271-b962-d4800e7ef390">